di_website/templates/publications/publication_page.html

```diff
@@ -31,12 +31,28 @@
                         {% endif %}
                         <h2 class="chapter-block__heading">{{ item.title }}</h2>
                         <p class="para {% cycle '' 'reverse-text' %}">{{ item | content_excerpt }}</p>
-                        <a href="{{ item.url }}" class="button button--a11y {% cycle '' 'button--reverse' %}" title="Read {{ item.label }}"><span>Read {{ item.label }}</span></a>
+                        <a href="{{ item.url }}" class="button button--a11y {% cycle '' 'button--reverse' %}" title="Read {% if item.is_foreword %}
+                                {{ item.button_titles }}
+                            {% else %}
+                                {{ item.label }}
+                            {% endif %}">
+                          <span>
+                            {% if item.is_foreword %}
+                                {{ item.button_titles }}
+                            {% else %}
+                                {{ item.label }}
+                            {% endif %}
+                          </span></a>
```

Comment:
Finding it hard to make sense of this ... probably needs some indenting, though I think it'll be better on one line, however long.

Reply:
Updated cached property to use the `item.label`